tests/unit/async_db/test_connection.py

```diff
@@ -18,6 +18,14 @@
 from firebolt.utils.token_storage import TokenSecureStorage
 
 
+async def test_connection_attributes(connection: Connection) -> None:
```

Comment:
Please add a comment on why do we need this test

Reply:
Yeah, I didn't have a chance to properly test this one so it looks odd. Reworked it and marked to skip since __slots__ has no effect in Connection class. :(